cpp/tests/groupby/sort/sql/single_column_sum.cu

```diff
@@ -0,0 +1,161 @@
+/*
```

Comment:
Likewise, don't refer to including null keys as `SQL` behavior---it doesn't mean anything to anyone who doesn't know about SQL. The description should be able to stand on its own.